src/parse.ts

```diff
@@ -76,12 +80,21 @@ export function parse(messageDefinition: string, options: ParseOptions = {}): Me
       : buildType(definitionLines, ROS1_GRAMMAR),
   );
 
+  // Filter out duplicate types.
+  // This will avoid that searching a type by name will return more than one result.
```

Comment:
Would be helpful to add some historical context about `mcap convert`, to explain why we shouldn't delete this code